libs/input-mapping/tests/ReaderTest.php

```diff
@@ -156,12 +118,13 @@ public function testReadTablesDefaultBackend(): void
         );
     }
 
+    #[NeedsDevBranch]
     public function testReadTablesDefaultBackendBranchRewrite(): void
     {
         file_put_contents($this->temp->getTmpFolder() . 'data.csv', "foo,bar\n1,2");
         $csvFile = new CsvFile($this->temp->getTmpFolder() . 'data.csv');
 
-        $clientWrapper = $this->getClientWrapper(null);
+        $clientWrapper = $this->clientWrapper;
```

Comment:
toto je právě u těch paralellních testů dost blbý spolíhat na $this->clientWrapper

Reply:
Upravil jsem testy, ktere pouzivaji veci z AbstractTestCase, nebo ktere jsem upravoval ja, tak aby teda kazdy test si zakladal clienta.